backend/ibutsu_server/db/upgrades.py

```diff
@@ -7,7 +7,7 @@
 from ibutsu_server.db.base import Boolean, Column, ForeignKey, Text
 from ibutsu_server.db.types import PortableUUID
 
-__version__ = 5
+__version__ = 7
```

Comment:
Version 6 has been deployed to production, not sure why this line was not set to version 6.